crates/node/service/src/actors/engine.rs

```diff
@@ -203,6 +213,21 @@ impl NodeActor for EngineActor {
                           let sent = self.engine_l2_safe_head_tx.send_if_modified(update);
                           trace!(target: "engine", ?sent, "Attempted L2 Safe Head Update");
                         }
+                        Err(EngineTaskError::Reset(e)) => {
+                            warn!(target: "engine", ?e, "Received reset request");
```

Comment:
```suggestion
                        Err(EngineTaskError::Reset(err)) => {
                            warn!(target: "engine", ?err, "Received reset request");
```
thinks they key `err` makes logs a bit better readable than `e` for newly onboarded, but no biggie